code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -13,10 +13,12 @@
 	shuttle_id = ""
 	var/leaders = 0
 
-/datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc)
+/datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc, announce_dispatch_message)
 	var/datum/squad/marine/cryo/cryo_squad = RoleAuthority.squads_by_type[/datum/squad/marine/cryo]
 	leaders = cryo_squad.num_leaders
-	return ..()
+	var/previous_members = length(members)
+	. = ..()
+	shipwide_ai_announcement("Successfully deployed [length(members) - previous_members] Foxtrot marines.")
```

Comment:
This can announce successfully deploying 0 marines ever prob_min was not reached:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/2d3da62f-1800-4915-abb5-e9352973a1a9)

Also I recommend considering setting the third argument `sound_to_play` to null and change the logic in `/proc/announcement_helper(message, title, list/targets, sound_to_play)` to handle this possibly being null and just opt to not play a sound in this situation rather than returning. It sounds funky for two sfx to play simultaneously (both sound/AI/distressreceived.ogg from `/datum/emergency_call/proc/spawn_candidates` and here as sound/misc/interference.ogg). Or implement logic to alter the distress beacon alert to include the count.

Reply:
Your branch is likely outdated.

The sound/AI/distressreceived.ogg from spawn_candidates is disabled on master.